src/charm.py

```diff
@@ -138,13 +144,13 @@ def _on_pebble_ready(self, event) -> None:
         with open("src/identity.default.schema.json", encoding="utf-8") as schema_file:
             schema = schema_file.read()
             self._container.push(self._identity_schema_file_path, schema, make_dirs=True)
+        self._container.push(self._config_file_path, self._config, make_dirs=True)
```

Comment:
The config needs dsn with database details to be able to start the service. Does it make sense to push it when database is not yet created?

Reply:
It was a compromise made earlier with Nikos' suggestion. I prefer to only push if database is created. I think this is also an okay approach.

Reply:
It was just a suggestion to make the flow simpler. If you prefer the previous approach I have no objections.

Reply:
It's ok, I also ended up using this approach in hydra.